index.html

```diff
@@ -61,6 +61,324 @@ <h1 class="display-5 fw-bold primary-text">I’m Yannick<br> Glad to see you!</h
       </div>
   </section>
 
+  <section class="work-section section-pd-50-50 section-mg-50-0">
+
+     <div class="container">
+
+        <div class="flex-container">
+          <div class="flex-item-left">
+              
+              <img src="image/portfolio1.png" class="img-responsive" alt="image-work1"></div>
+          
+          <div class="flex-item-right">
+              <h2 class="work-title">Tonic</h2>
+              <br>
+              <br>
+              <div class="work-detail">
+                <div class="work-info">
+                 <ul>
+                   <li class="intro">Canopy</li>
+                   <li> <i class="fa fa-circle"></i> Back End Dev</li>
+                   <li><i class="fa fa-circle"></i> 2015</li>
+                 </ul>
+                 </div>
+
+                 <p class="description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                 </p>
+
+                 <div class="work-tech">
+                 <ul>
+                   <li>html</li>
+                   <li>css</li>
+                   <li>javascript</li>
+                 </ul>
+                 </div>
+                <div>
+              </div>
+              </div>
+              
+              <button class="btn btn-one" type="button">See Project</button>
+          </div>
+        </div>
+     
+     </div>
+  
+  </section>
+      
+      
+   <section class="work-section section-pd-50-50">
+
+     <div class="container">
+
+        <div class="flex-container">
+          <div class="flex-item-left work22">
+              
+            <h2 class="work-title">Multi-Post Stories</h2>
+              <br>
+              <br>
+              <div class="work-detail">
+                <div class="work-info">
+                 <ul>
+                   <li class="intro">FACEBOOK</li>
+                   <li> <i class="fa fa-circle"></i> Full Stack Dev</li>
```

Comment:
- [ ] The list of projects should use a ul tag to group them, and li per project. Not `section` tags as you used but `li` tag for each project all wrapped in a `ul` tag.

Reply:
Hi Please can you explain me this better cause I dont really understand